src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -109,22 +109,23 @@ WMTS_Provider.prototype.url = function url(coWMTS, layer) {
  * @param {type} coWMTS : coord WMTS
  * @returns {WMTS_Provider_L15.WMTS_Provider.prototype@pro;_IoDriver@call;read@call;then}
  */
-WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, parameters) {
-    var cooWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
+WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer) {
+    var coordWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
     var pitch = new THREE.Vector3(0.0, 0.0, 1.0);
 
-    if (parameters.ancestor) {
-        // account for possible level offset between coords and tile level
-        // (e.g for PM texture cooWMTS.level = tile.level + 1)
-        var levelOffset = cooWMTS.zoom - tile.level;
+    const searchInParent = tile.materials[0].getElevationLayerLevel() < 0 || coordWMTS.zoom > layer.zoom.max;
 
-        cooWMTS = this.projection.WMTS_WGS84Parent(
-            cooWMTS,
-            this.computeLevelToDownload(tile, layer, parameters.ancestor.level) + levelOffset,
-            pitch);
+    if (searchInParent) {
+        const parentZoom = tile.parent.materials[0].getElevationLayerLevel();
+        if (parentZoom > -1) {
+            coordWMTS = this.projection.WMTS_WGS84Parent(
+                coordWMTS,
+                Math.max(layer.zoom.min, parentZoom),
+                pitch);
+        }
     }
 
-    var url = this.url(cooWMTS, layer);
+    var url = this.url(coordWMTS, layer);
```

Comment:
When `searchInParent` is true you could skip the `this.url()` call and directly use parent's texture.

If this is done on all providers, we could remove their usage of `CacheResources` which IMHO is a good idea:
* the browser already cache network request
* we never evict data from this cache

This would allow to effectively delete unused THREE.Texture which is currently not done (even after calling material.dispose() because CacheResource still own a reference).